pkg/management/rbac.go

```diff
@@ -40,6 +61,28 @@ func (s *Server) CreateRoleBinding(ctx context.Context, in *corev1.RoleBinding)
 	return &emptypb.Empty{}, s.coreDataSource.StorageBackend().CreateRoleBinding(ctx, in)
 }
 
+func (s *Server) UpdateRoleBinding(
+	ctx context.Context,
+	in *corev1.RoleBinding,
+) (*emptypb.Empty, error) {
+	if err := validation.Validate(in); err != nil {
+		return &emptypb.Empty{}, err
+	}
+
+	oldRb, err := s.GetRoleBinding(ctx, in.Reference())
+	if err != nil {
+		return &emptypb.Empty{}, err
+	}
+
+	_, err = s.coreDataSource.StorageBackend().UpdateRoleBinding(ctx, oldRb.Reference(), func(rb *corev1.RoleBinding) {
+		rb.RoleId = in.GetRoleId()
+		rb.Subjects = in.GetSubjects()
+		rb.Taints = in.GetTaints()
```

Comment:
Can't store the taints field, it's read-only. Might also want to validate it in the implementation of UpdateRoleBinding as well to throw an error if it's non-nil

Reply:
working on validations now